go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/hashicorp/terraform-plugin-sdk/v2 v2.34.0
 	github.com/stretchr/testify v1.9.0
 	github.com/tailscale/hujson v0.0.0-20221223112325-20486734a56a
-	github.com/tailscale/tailscale-client-go v1.17.0
+	github.com/tailscale/tailscale-client-go v1.17.1-0.20240718200212-ff9b01c0d472
```

Comment:
TODO: this needs to be updated to the released version once https://github.com/tailscale/tailscale-client-go/pull/84 is through